lib/route_types.pl

```diff
@@ -121,7 +121,10 @@ sub codelist {
 	}
 	$str =~ s/ $//;
 	push @lines, $str . "\\n\" \\\n";
-	push @lines, "  \"       > - selected route, * - FIB route, q - queued, r - rejected, b - backup\\n\\n\"";
+	push @lines, "  \"       > - selected route, * - FIB route, q - queued, r - rejected, b - backup\\n\"";
+	push @lines, "  \"       t - trapped, o - offload failure\\n\"";
```

Comment:
the text says "offload failure", but the sense of the flags seems to be "offloaded" ?

Reply:
yep that is why I test the negative of it below